decoder/DecoderApi_unittest.cpp

```diff
@@ -283,6 +291,12 @@ TEST_P(DecodeApiTest, Format_Change)
     while (frames.getFrame(buffer, info)) {
 
         YamiStatus status = decoder->decode(&buffer);
+        if (YAMI_UNSUPPORTED == status) {
```

Comment:
duplicate code here. 
please use
bool checkStatus(YamiStatus status)
to  remove all duplicates